docs-compose/overwatch_docs/content/DataEngineer/Productionizing.md

```diff
@@ -55,6 +56,240 @@ created each run but it will be replaced on each subsequent run. If you would li
+
+1. "snapshotRootPath/data" - Contains the backup data of Bronze Tables.
+2. "snapshotRootPath/clone_report" - Contains the data containing run metrics.
+3. "snapshotRootPath/checkpoint" - Checkpoint Location for Streaming Operation.
+
+#### How to run the Snapshot Process
+Currently, Users need to run the snapshot process through 
+1. Databricks Job.
+2. Notebook.
+
+##### Snapshot through Databricks Notebook
+
+The code snippet to run the Snapshot process is as below:
+
+```scala
+import com.databricks.labs.overwatch.pipeline.Snapshot
+val sourceETLDB = "ow_bronze_snapshot_etl"
+val targetPrefix = "/mnt/overwatch_playground/721-snap/incremental"
+val pipeline = "Bronze,Silver,Gold"
+val snapshotType = "Incremental"
+val tablesToExclude = "notebook_silver,notebook_gold"
+Snapshot.process(sourceETLDB,targetPrefix,snapshotType)
+```
+
+We need below parameters to run the snapshot process through Databricks Notebook:
+
+| Param           | Type   | Optional | Default Value        | Description                                                                                                                                        |
+|-----------------|--------|----------|----------------------|----------------------------------------------------------------------------------------------------------------------------------------------------|
+| sourceETLDB     | String | No       | NA                   | Source Database Name for which Snapshot need to be done.                                                                                           |
+| targetPrefix    | String | No       | NA                   | Target path where Snapshot need to be done.                                                                                                        |
+| snapshotType    | String | No       | NA                   | Type of Snapshot to be performed. "Full" for Full Snapshot, "Incremental" for Incremental Snapshot                                                 |
+| pipeline        | String | Yes      | "Bronze,Silver,Gold" | Define the Medallion Layers. Argument should be in form of "Bronze, Silver, Gold"(All 3 or any combination of them)                                |
+| cloneLevel      | String | Yes      | "DEEP"               | Clone Level for Snapshot. By Default it is "DEEP". You can also specify "SHALLOW" Clone                                                            |
+| tablesToExclude | String | Yes      | ""                   | Array of table names to exclude from the snapshot. Tables should be separated with ",". This is the table name only - without the database prefix. |
+
+After snapshot process is done the tables from Source ETL Database will be cloned in the target prefix location. The structure for target prefix location would be like below:
+
+![Snapshot_Target](/images/DataEngineer/snapshotTarget.png)
+
+All the tables data would be there in "targetPrefix/data" folder as shown below:
```

Comment:
Change to: "The snapshotted data will be in the "targetPrefix/data" path as shown below:"

Reply:
Done